static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js

```diff
@@ -22,7 +26,7 @@ define([
         if (this.$adSlot.data('name') !== 'right') {
             return;
         }
-        articleBodyOffset = qwery('.content__article-body')[0].offsetTop;
+        articleBodyOffset = config.page.hasShowcaseMainPicture ? $('.media-primary').dim().height : qwery('.content__article-body')[0].offsetTop;
```

Comment:
This should probably be `fastdom`ed, as `dim()` and `offsetTop` will trigger a layout, and you're writing immediately after.